statement.go

```diff
@@ -15,6 +15,39 @@ import (
 	"unsafe"
 )
 
+type StmtType C.duckdb_statement_type
+
+const (
+	DUCKDB_STATEMENT_TYPE_INVALID      StmtType = C.DUCKDB_STATEMENT_TYPE_INVALID
```

Comment:
Let's make this consistent with the types in `type.go`. I.e, let's not repeat the package name `DUCKDB_`:
```Go
STATEMENT_TYPE_INVALID      StmtType = C.DUCKDB_STATEMENT_TYPE_INVALID
```

Reply:
Done